searchAdressMap.js

```diff
@@ -0,0 +1,30 @@
+function initSeachMap() {
```

Comment:
looks like the same function that `initMap` is, but with some different values?

how about merging it into one function that receives arguments?